packages/@okta/vuepress-site/docs/release-notes/2020-09-3/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.09.3
+
+| Change                                                    | Expected in Preview Orgs |
+| --------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2020.09.3](#bugs-fixed-in-2020-09-3)       | September 23, 2020           |
+
+### Bugs fixed in 2020.09.3
+
+* If a user was converted to use an [external Federated IdP instead of Okta](/docs/reference/api/users/#request-example-convert-a-user-to-a-federated-user), any subsequent attempt to convert the user with a call to the `/users/${userId}/lifecycle/reset_password` endpoint returned an HTTP 501 error instead of an HTTP 400 error. (OKTA-323343)
```

Comment:
Really shouldn't have this bulleted since it's the only item. I typically don't bullet when I do the Release Notes and there is only one bug. However, I notice that there isn't consistency with that when I look at what we have out there. But, that's what I've been trying to do, because based on the Microsoft Style Guide, you don't bullet just one item.